src/components/Features/Features.css

```diff
@@ -0,0 +1,92 @@
+.features {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 62px 50px;
+  gap: 10px;
+}
+
+.featureHead {
+  color: #ffffff;
+}
+
+.head {
+  text-align: center;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: 0.2px;
+}
+
+.featuresHeader {
+  width: 100%;
+  padding: 10px;
+}
+
+.featureSubHead .featureText {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 30px;
+  text-align: center;
+  letter-spacing: 0.2px;
+}
```

Comment:
the same goes for this one apart from the 
`font-size` and `text-align` the rest of the properties 
are not necessary. also you didn't really need to put this
in a `<span>` apply the CSS directly on the `<p>` tag

Tip: for better control, you could wrap the 
feature-section heading and description into one div
then apply `margin-bottom` to separate this block from the frame-block